test_main.py

```diff
@@ -3,97 +3,44 @@
 # To run the tests, use: python3 -m pytest --capture=sys
 
 from database import Database
+from filters import Filter
+import filters
 from main import main
 from test_utils import region_dict
+import random
 
 db = Database()
 print(len(db))
 
 
-def test_no_args(capsys):
```

Comment:
This one is obsolete now because the new command engine does works with no arguments.